### ISSUE TITLE
Fix background color on dropdown

### ISSUE THREAD

**Issue by ricardoom** [NYPL/design-toolkit#269]
Green used for selected filters has insufficient color contrast of 3.91 (foreground color: #008918, background color: #efedeb, font size: 12.0pt, font weight: normal).

**Comment by ricardoom** [comment]
New Color found here:

http://contrast-finder.tanaguru.com/result.html?foreground=%23008918&background=%23FEFDFD&ratio=4.5&isBackgroundTested=true&algo=Rgb
